app/views/supervisor/dashboard/index.html.slim

```diff
@@ -1,30 +1,43 @@
 h2 Supervisor Dashboard
-h5 Alpha Version
 
 div.row
-  div.dashboard-box.col-sm-12
+  div.dashboard-box.col-xs-12
     div.background-color
       section.team-title
         h3 Overview
-        h5 Under Construction
       section.dashboard
-        p = link_to "Supervisors on GitHub", 'https://github.com/rails-girls-summer-of-code/supervisors'
-        p = link_to "Guide", 'https://github.com/rails-girls-summer-of-code/supervisors/blob/master/guide.md'
-        /this is where all the aggregated messages and info should go
-        /= flash[:notice]
+        div.row
+          div.links.col-sm-6 class=("col-lg-8" if supervised_teams.count > 2)
+            p = link_to "Guide", 'https://github.com/rails-girls-summer-of-code/supervisors/blob/master/guide.md'
+            p = link_to "Supervisors on GitHub", 'https://github.com/rails-girls-summer-of-code/supervisors'
+            table.legend
+              tr
+                td
+                  .performance-icon = render 'green'
+                td
+                  '= Alive and kicking
+              tr
+                td
+                  .performance-icon = render 'orange'
+                td
+                  '= Needs Tender Loving Care
+              tr
+                td
+                  .performance-icon = render 'red'
+                td
+                  '= Red alert
+          div.col-sm-6 class=("col-lg-4 padding" if supervised_teams.count > 2)
+            = simple_form_for @current_user.notepad, url: supervisor_notes_path do |f|
```

Comment:
It's better to access the `@current_user` w/o the @, ie. via its corresponding helper method.